#!/bin/bash
#use: creates new columns for the first (mouse type), second (transcript type), and third (number hits) rows of each set. Then names specific to HMMsearchHits.txt are translated to protein names. Finally, a file with protein abbreviations is merged to the file.
awk '{printf "%s%s",$0,NR%3?",":RS}' HMMsearchHits.txt | sed -e s/protein.fasta//g | sed -e s/_Proteins.fasta.aligned.hmm//g | sed 's/'Transcript10'/'synaptoporin'/g' | sed 's/'Transcript09'/'LIM_homeobox_protein_2'/g' | sed 's/'Transcript08'/'ATPase'/g' | sed 's/'Transcript06'/'tyrosine_phosphatase'/g' | sed 's/'Transcript02'/'asc-type_AA_transporter_2'/g' | sed 's/'Transcript01'/'glutathione_S_transferase'/g' > hitTableFinalprotein.csv
paste -d',' hitTableFinalprotein.csv GeneNames.csv > hitTableFinalprotAbbreviated.csv
